pkg/printers/printers.go

```diff
@@ -115,7 +115,7 @@ func GetStandardPrinter(typer runtime.ObjectTyper, encoder runtime.Encoder, deco
 		}
 
 	case "wide", "":
-		humanPrintFlags := NewHumanPrintFlags(options.Kind, options.NoHeaders, options.WithNamespace, options.AbsoluteTimestamps)
+		humanPrintFlags := NewHumanPrintFlags(options.NoHeaders, options.WithNamespace, options.AbsoluteTimestamps)
```

Comment:
This is actually not hit anymore, since get.go asks for the printer directly from the flags. Will remove